app/index.js

```diff
@@ -16,7 +16,7 @@ updateNotifier();
 
 // Grab a list of all CF components, we'll use it later.
 var components = request({
-  uri: 'https://api.github.com/orgs/cfpb/repos?per_page=100',
+  uri: 'https://api.github.com/search/repositories?q=%22cf-%22+NOT+cfpb+NOT+cfgov+NOT+deprecated+user:cfpb+language:css',
```

Comment:
You can, I was just trying to keep the query string down in size, since it's kid of long already. It's probably better to just future-proof it now though. Should we add `html` to [cover our bases](https://api.github.com/search/repositories?q=%22cf-%22+NOT+cfpb+NOT+cfgov+NOT+deprecated+user:cfpb+language:JavaScript+language:css+language:html).